app/views/templates/Layout.scala.html

```diff
@@ -45,49 +45,51 @@
         dutyDeferment: Boolean = false
 )(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
 
-    @phaseBannerContent = {
-    @messages("feedback.before")
-        <a class='govuk-link' href='/contact/beta-feedback-unauthenticated?service=CDS-FIN'>@messages("feedback.link")</a>
-    @messages("feedback.after")
-    }
+@phaseBannerContent = {
+@messages("feedback.before")
```

Comment:
if messages is inside phaseBannerContent do we not want to indent it once?

@phaseBannerContent = {
    @messages("feedback.before")
        <a class='govuk-link' href='/contact/beta-feedback-unauthenticated?service=CDS-FIN'>@messages("feedback.link")</a>
    @messages("feedback.after")
}

Reply:
This was an unintentional change, have restored.